contrib/schemas/rule.json

```diff
@@ -0,0 +1,105 @@
+{
+    "additionalProperties": false, 
+    "type": "object", 
+    "properties": {
+        "uid": {
+            "type": "string"
+        }, 
+        "tags": {
+            "items": {
+                "type": "object"
+            }, 
+            "type": "array", 
+            "description": "User associated metadata assigned to this object."
+        }, 
+        "metadata_file": {
+            "default": "", 
+            "type": "string", 
+            "description": "Path to the metadata file relative to the pack directory."
+        }, 
+        "id": {
+            "default": null, 
+            "type": "string"
+        }, 
+        "description": {
+            "type": "string"
+        }, 
+        "name": {
+            "required": true, 
+            "type": "string"
+        }, 
+        "ref": {
+            "type": "string", 
+            "description": "System computed user friendly reference for the action.                                 Provided value will be overridden by computed value."
```

Comment:
Weird spaces added here.

Reply:
Good catch. No idea. Must be from the formatting in the content models.  I'll fix it.

Reply:
Awesome, thanks! Also, I think the description here was copy/pasted from `st2common/st2common/models/api/action.py`, and the `ref` value wasn't changed. I think this should be changed to:

> System computed user friendly reference for the **rule**. Provided value will be overridden by computed value.

(Emphasis mine)

Reply:
I'm not 100% sure about that though. It could be referring to the action the rule references.

Reply:
Good catch. Fixed.